extensions/amp-story-player/validator-amp-story-player.protoascii

```diff
@@ -42,4 +42,40 @@ descendant_tag_list {
   tag: "A"
   tag: "SPAN"
   tag: "I-AMPHTML-SIZER"  # Only allowed when document is transformed AMP
+  tag: "IMG"
+}
+tags: {
+  html_format: AMP
+  tag_name: "IMG"
+  spec_name: "AMP-STORY-PLAYER IMG"
+  mandatory_ancestor: "AMP-STORY-PLAYER"
+  attrs: {
+    name: "loading"
+    value: "lazy"
+  }
+  attrs: { name: "data-amp-story-player-poster-img" }
```

Comment:
Is this required? None of your examples have a value set, but the rules allow for any value. Any guidance on when to use this attribute, when not, and what allowed are the supported values?